cloudify_cli/async_commands/audit_log.py

```diff
@@ -0,0 +1,175 @@
+import asyncio
+import json
+
+import click
+
+from ..cli import cfy, helptexts
+from ..exceptions import CloudifyCliError
+from ..logger import get_global_json_output
+from ..table import print_data
+from ..utils import before_to_utc_timestamp
+
+AUDITLOG_COLUMNS = ['ref_table', 'ref_id', 'operation', 'creator_name',
+                    'execution_id', 'created_at']
+
+
+@cfy.group(name='auditlog')
+@cfy.assert_manager_active()
+def auditlog():
+    """Manage the audit log"""
+    pass
+
+
+@auditlog.command(name='list',
+                  short_help='List audit log entries')
+@click.option('-c', '--creator-name',
+              help=helptexts.AUDIT_CREATOR_NAME)
+@click.option('-e', '--execution-id',
+              help=helptexts.AUDIT_EXECUTION_ID)
+@click.option('-i', '--since',
+              help=helptexts.AUDIT_SINCE)
+@click.option('-f', '--follow', '--tail',
+              help=helptexts.AUDIT_FOLLOW,
+              is_flag=True)
+@cfy.options.timeout(default=300)
+@cfy.options.sort_by()
+@cfy.options.descending
+@cfy.options.pagination_offset
+@cfy.options.pagination_size
+@cfy.options.common_options
+@cfy.pass_logger
+@cfy.pass_client()
+def list_logs(creator_name,
+              execution_id,
+              since,
+              follow,
+              timeout,
+              sort_by,
+              descending,
+              pagination_offset,
+              pagination_size,
+              logger,
+              client,
+              ):
+    since_timestamp = before_to_utc_timestamp(since) if since else None
+    if follow:
+        loop = asyncio.get_event_loop()
+        if not loop:
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
```

Comment:
I don't think this is necessary? shouldn't .get_event_loop() just create one?

Reply:
Oh, in deed it does